language/move-analyzer/src/symbols.rs

```diff
@@ -333,7 +333,9 @@ impl SymbolicatorRunner {
         let thread_mtx_cvar = mtx_cvar.clone();
         let runner = SymbolicatorRunner { mtx_cvar };
 
-        thread::spawn(move || {
+        thread::Builder::new()
+            .stack_size(16 * 1024 * 1024) // building Move code requires a larger stack size on Windows
```

Comment:
bit magical, an reasoning behind the size?

Reply:
It has been chosen somewhat arbitrarily, based on experience with using the compiler in a Move adapter.